abdulfataiaka/pluralsight-admin
# #161284348 Update readme file

#### What does this PR do
- Updates readme file

#### Description of work done
- Include application details
- Include deployment details
- Include license and author details

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to the branch `ch-update-readme-161284348`
- Check ReadMe file on Github

#### Relevant pivotal tracker story
[161284348](https://www.pivotaltracker.com/story/show/161284348)
